src/main/java/net/tiagofar78/prisonescape/items/TrapItem.java

```diff
@@ -28,12 +35,30 @@ public boolean isIllegal() {
 
     @Override
     public Material getMaterial() {
-        return Material.COBWEB;
+        return Material.POWERED_RAIL;
     }
 
     @Override
     public boolean isBuyable() {
         return true;
     }
 
+    @Override
+    public void use(PlayerInteractEvent e) {
+        Player player = e.getPlayer();
+        String playerName = player.getName();
+
+        Location loc = e.getClickedBlock().getLocation();
```

Comment:
I think it would be better if the trap was placed where players are instead of where they click. If the ceiling doesn't have a block above itself you would be putting the trap above the roof.